src/widget/form/settings/generalform.cpp

```diff
@@ -109,6 +110,8 @@ GeneralForm::GeneralForm(SettingsWidget *myParent) :
     bodyUI = new Ui::GeneralSettings;
     bodyUI->setupUi(this);
 
+    Settings& s = Settings::getInstance();
```

Comment:
would be neat to make the choice between `s` or `Settings::getInstance()` consistent throughout the method.